maven-scm-plugin/src/main/java/org/apache/maven/scm/plugin/AbstractScmMojo.java

```diff
@@ -445,7 +446,7 @@ public void checkResult( ScmResult result )
             getLog().error( result.getCommandOutput() == null ? "" : result.getCommandOutput() );
 
             throw new MojoExecutionException(
-                "Command failed." + StringUtils.defaultString( result.getProviderMessage() ) );
```

Comment:
The full stop should be replaced by a colon followed by a space.

Reply:
done